### PR TITLE
Added case and space-replace prop to tags interface

### PR DESCRIPTION
**Pull request by Nitwel**
grafik

### CODE REVIEW

**Comment by rijkvanzanten** [comment]
@benhaynes Thoughts on the "replace space" option? I feel like we might want to change it to "db safe" so it matches a little closer to what we're doing elsewhere (force lowercase + underscores)

**Comment by benhaynes** [comment]
I like the granularity of this... since it's not really about being DB safe, and more about having consistent tag formatting for your specific use-case. Although I would change the label to `Whitespace` and then have the action in the dropdown:

* `Leave as-in`
* `Replace with underscore`
* `Replace with hyphen`

Also, let's use `capitalization` instead of `case`.

Lastly, let's make sure OUR capitalization is consistent for all these options. For instance:

* `Force Alphabetical Order`
* `Allow other values`

File: app/src/lang/en-US/interfaces.json [hunk 1]
```diff
@@ -152,8 +152,14 @@
 		"tags": {
 			"tags": "Tags",
 			"description": "Select or add tags",
-			"lowercase": "Lowercase",
-			"lowercase_label": "Force Lowercase",
+			"space_replace": "Whitespace",
```
Comment (rijkvanzanten) [inline app/src/lang/en-US/interfaces.json:155]: Can you keep the translations key inline with the actual value? 

`space_replace` shouldn't be rendered as `Whitespace`. `space_replace` = `Space Replace`, `whitespace` = `Whitespace`

File: app/src/interfaces/tags/tags.vue [hunk 2]
```diff
@@ -160,9 +177,9 @@ export default defineComponent({
 		}
 
 		function removeTag(tag: string) {
-			selectedValsLocal.value = selectedValsLocal.value.filter(
-				(savedTag) => (props.lowercase ? savedTag.toLowerCase() : savedTag) !== tag
-			);
+			console.log(tag);
```
Comment (rijkvanzanten) [inline app/src/interfaces/tags/tags.vue:180]: Found one!

**Review by rijkvanzanten — APPROVED** [review]

**Comment by rijkvanzanten** [comment]
@Nitwel Lets clear out that stray console log, and then we're good to merge 👍

File: app/src/interfaces/tags/tags.vue (hunk 2 above)
Comment (Nitwel) [reply app/src/interfaces/tags/tags.vue:180]: Upsie!

**Pull request merged**